Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs

```diff
@@ -85,6 +85,14 @@ public bool PlayerInShelter
             }
         }
 
+        public bool PlayerInAncientShelter
+        {
+            get
+            {
```

Comment:
May want to do either do a safety check in game to ensure non-MSC won't break here or wrap this in ModManager.MSC

Reply:
imo it's safe as is, it's a bool that defaults to false

Reply:
Works for me